package.json

```diff
@@ -27,8 +27,8 @@
     "test:keepdb": "dotenv -e .env.test -- jest --runInBand",
     "test": "dotenv -e .env.test -- prisma migrate reset --force --skip-generate > /dev/null && yarn test:keepdb",
     "db:client:generate": "prisma generate",
-    "db:migrate": "prisma migrate dev",
-    "db:client:format": "prisma format"
+    "db:client:format": "prisma format",
```

Comment:
Should `master` be rebased into this branch?

Reply:
I already rebased, I think I flipped order is all (had created PR for format after this one).